### PR TITLE
Send to bigquery

### PR DESCRIPTION
**Pull request by p-lu**
# Description
* Adds a step to send the results of classification to BigQuery
* Moved the BigQuery client files into `src/`

### CODE REVIEW

**Review by roisin-howard — APPROVED** [review]
LGTM! 🚀

File: src/recycle_damn_iot.py [hunk 1]
```diff
@@ -75,6 +79,18 @@ def parse_result(result):
             percentage = str("{:.2f}".format(percentage))
     return percentage
 
+def send_result(label, percentage):
+    bq = BigQueryClient()
+    
+    pred = Prediction()
+    pred.set_id(123)
```
Comment (markobrien-wpengine) [inline src/recycle_damn_iot.py]: Is this always going to be static?

**Review by markobrien-wpengine — CHANGES_REQUESTED** [review]

File: src/recycle_damn_iot.py (hunk 1 above)
Comment (p-lu) [reply src/recycle_damn_iot.py]: Change incoming to use a timestamp instead

Comment (p-lu) [reply src/recycle_damn_iot.py]: Done

**Review by markobrien-wpengine — APPROVED** [review]
LGTM

**Pull request merged**